react/modules/tasks/redux.md

```diff
@@ -0,0 +1,58 @@
+# Redux. Redux Toolkit, RTK Query. Context api.
+
+## Technical requirements.
+
+1. Create a separate branch for this task from the previous branch task.
+2. Redux Integration:
+   - Integrate Redux into your application. You'll need to set up the Redux store and reducers using Redux Toolkit.
+3. RTK Query integration:
+   - RTK Query Implementation: Use Redux Toolkit Query (RTK Query) to make API calls and cache the results. This will modify your previous API call implementation;
+   - Use RTK Query to show whether the data is being loaded. Avoid prop-drilling, if it is required to show indicator in child component, put the respective flag to the store;
```

Comment:
if it is required to show an indicator in a child component, put the respective flag in the store;

Reply:
done